### PR TITLE
Add documentation on routing in Weblocks

### PR DESCRIPTION
**Pull request by svetlyak40wt**
This should close issue https://github.com/40ants/weblocks/issues/28

### CODE REVIEW

File: docs/source/components.rst [hunk 1]
```diff
@@ -2,6 +2,9 @@
  Components of Weblocks
 ========================
 
+.. warning:: This section is outdated and belong to the old Weblocks documention.
```
Comment (vindarel) [inline docs/source/components.rst]: belong/s/

File: docs/source/contributing.rst [hunk 2]
```diff
@@ -2,16 +2,24 @@
...
+
+5. Ensure you've updated documentation in the ``docs`` folder if you've
+   changed or added functionality.
+6. Describe your changes for in a readable way ontop of the
+   ``ChangeLog.rst``. Increment a version number according to `semver`_.
+   Also, update a ``version.lisp-expr`` file.
+7. Make a pull-request and make the world a better place!
+
 
+.. _semver: http????
```
Comment (vindarel) [inline docs/source/contributing.rst]: seems https://semver.org/

File: docs/source/dependencies.rst [hunk 3]
```diff
@@ -2,6 +2,10 @@
  Dependency handling
 =====================
 
+.. warning:: This section is outdated and belong to the old Weblocks documention.
+
+             SHOULD BE REWRITTEN.
+
```
Comment (vindarel) [inline docs/source/dependencies.rst:8]: very useful to know!

File: docs/source/installation.rst [hunk 4]
```diff
@@ -47,9 +47,34 @@ For a basic comfortable SBCL setup, see :ref:`Basic SBCL setup`.
  
 
 Installation
-------------
+============
 
-There is a comprehensive installation guide on the main web site:
+.. note:: Official quicklisp distribution contains an old Weblocks
```
Comment (vindarel) [inline docs/source/installation.rst]: /The/ official

about the newer

**Comment by vindarel** [comment]
'lo, I encourage you to merge this, even without the route documentation. It is already less confusing to not see the outdated sections.

**Comment by vindarel** [comment]
Hello, encouragements to merge this and remove the outdated documentation!

**Comment by svetlyak40wt** [comment]
Thanks for grumbling, @vindarel!

File: docs/source/components.rst (hunk 1 above)
Comment (svetlyak40wt) [reply docs/source/components.rst]: 👍

File: docs/source/contributing.rst (hunk 2 above)
Comment (svetlyak40wt) [reply docs/source/contributing.rst]: 👍

File: docs/source/installation.rst (hunk 4 above)
Comment (svetlyak40wt) [reply docs/source/installation.rst]: 👍

**Comment by svetlyak40wt** [comment]
Merged!

**Comment by vindarel** [comment]
Congratulations :)

**Pull request merged**